wrappers/python/openmm/app/metadynamics.py

```diff
@@ -192,6 +192,12 @@ def getCollectiveVariables(self, simulation):
         """Get the current values of all collective variables in a Simulation."""
         return self._force.getCollectiveVariableValues(simulation.context)
 
+    def getHillHeight(self, simulation):
+        """Get the current height of the Gaussian hill in kJ/mol"""
+        energy = simulation.context.getState(getEnergy=True, groups={31}).getPotentialEnergy()
```

Comment:
The force group should not be hard-coded:
```python
forceGroup = self._force.getForceGroup()
energy = simulation.context.getState(getEnergy=True, groups={forceGroup}).getPotentialEnergy()
```
There was already a similar bug (https://github.com/openmm/openmm/pull/3060).

Reply:
Yeah, that's a good catch.